week01/hyeni/src/script.ts

```diff
@@ -0,0 +1,95 @@
+// 1. HTML 요소 선택 (핸드북 자바스크립트 편 참고)
+const todoInput = document.getElementById('todo-input') as HTMLInputElement;
+const todoForm = document.getElementById('todo-form') as HTMLFormElement;
+const todoList = document.getElementById('todo-list') as HTMLUListElement;
+const doneList = document.getElementById('done-list') as HTMLUListElement;
+
+// 2. 할 일이 어떻게 생긴 애인지 Type을 정의
+type Todo = {
+    id: number;
+    text: string;
+};
+
+let todos: Todo[] = [];
+let doneTasks: Todo[] = [];
+
+// - 할 일 목록 렌더링 하는 함수를 정의
+const renderTasks = (): void => {
+  todoList.innerHTML = '';
+  doneList.innerHTML = '';
+  
+  todos.forEach((todo) : void => {
+    const li = createTodoElement(todo, false);
+    todoList.appendChild(li);
+  });
+
+  doneTasks.forEach((todo) : void => {
+    const li = createTodoElement(todo, true);
+    doneList.appendChild(li);
+  });
+};
+
+// 3. 할 일 텍스트 입력 처리 함수
+const getTodoText = (): string => {
+  return todoInput.value.trim();
+};
+
+// 4. 할 일 추가 처리 함수
+const addTodo = (text: string): void => {
+  todos.push({ id: Date.now(), text });
+  todoInput.value = '';
+  renderTasks();
+}
+
+// 5. 할 일 상태 변경 (완료로 이동)
+const compleTask = (todo: Todo): void => {
```

Comment:
가독성 측면에서 의미가 명확한 completeTask로 변경하는 게 어떨까요?
```suggestion
const completeTask = (todo: Todo): void => {

Reply:
말씀 감사합니다! 리뷰 주신 부분 수정 완료하였습니다^_^